lib/sockunion.c

```diff
@@ -621,7 +604,8 @@ int sockunion_cmp(const union sockunion *su1, const union sockunion *su2)
 			return -1;
 	}
 	if (su1->sa.sa_family == AF_INET6)
-		return in6addr_cmp(&su1->sin6.sin6_addr, &su2->sin6.sin6_addr);
+		return memcmp(&su1->sin6.sin6_addr, &su2->sin6.sin6_addr,
```

Comment:
We have already IPV6_ADDR_CMP(), which is included in prefix.h (included in this file also).

Reply:
sure switched it over